client/src/GlobalStyles.js

```diff
@@ -81,6 +81,7 @@ function GlobalStyles() {
           height: 100vh;
           justify-content: center;
```

Comment:
I'm not a fan of setting too many specific global styles. In your mobile version, you might need the flexbox in your body, but what if you decide to change the layout for the desktop version? You have to deal with this global flexbox then, regardless if you need it or not. I recommend to only set very basic styles in global styles and add a main-container as flexbox.
This is of course just an opinion, nothing you have to follow along =)